test-utils/database-urls.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   DATABASE_URL_MSSQL: 'mssql://sa:forest2019:@localhost:1432/model',
   DATABASE_URL_MYSQL: 'mysql://forest:secret@localhost:8999/lumber-sequelize-test',
   DATABASE_URL_POSTGRESQL: 'postgres://forest:secret@localhost:54369/lumber-sequelize-test',
+  DATABASE_URL_POSTGRESQL_OLD: 'postgres://forest:secret@localhost:54368/lumber-sequelize-test',
```

Comment:
The only comment I will leave is here, not sure we should name it old as if we want to add another version it will be harder to distinguish them. May be we should add the version number in the variable name

Reply:
We could follow the Mongodb naming convention above ;)

Reply:
I understand your opinion guys. I started with version numbers in this file :) 
But after some usage, I found this was not useful, and maybe error-prone since this file can be not in sync with `docker-compose` and with the `multiple-database-version-helper.js::sqlDatabases`.

If it sounds for you, we can change to have only _OLD and _LAST suffixes in this file, and the file won't change except adding a new kind of DB.

Reply:
OK, easier to maintain without version number in the name.
But also less contextual when you read the variable name in the code.

Whatever the choice for consistency we should not live with 2 "conventions" (`_3_2` vs `_OLD`), so MongoDB variable names should be impacted by this new decision.

What about this:
```javascript
module.exports = {
  DATABASE_URL_MONGODB_MIN: 'mongodb://localhost:27015',
  DATABASE_URL_MONGODB_MAX: 'mongodb://localhost:27016',
  DATABASE_URL_MSSQL_MAX: 'mssql://sa:forest2019:@localhost:1432/model',
  DATABASE_URL_MYSQL_MAX: 'mysql://forest:secret@localhost:8999/lumber-sequelize-test',
  DATABASE_URL_POSTGRESQL_MIN: 'postgres://forest:secret@localhost:54368/lumber-sequelize-test',
  DATABASE_URL_POSTGRESQL_MAX: 'postgres://forest:secret@localhost:54369/lumber-sequelize-test',
};

```

Reply:
MIN/MAX looks better than OLD. Like that I won't think about my grandma at work :)

Reply:
It deserves a deeper look, but we should delete this file, and delete `multiple-database-version-helper.js::sqlDatabases` variable too. 
We may generate `sqlDatabases` directly from reading docker-compose.yml
A good way to have nothing written in multiple places, isn't it?